src/denoise/shader/poisson_denoise.frag

```diff
@@ -166,7 +166,7 @@ void main() {
 
   float roughnessRadius = mix(mat.roughness * mat.roughness, 1., specularPhi);
 
-  float r = flatness * radius * exp(-max(inputs[0].a, inputs[1].a) * 0.01);
```

Comment:
I saw this was added but the inputs[0].a isn't guaranteed defined at this point and I was too lazy to figure out why. So I just removed this change for now.